docs/modules/reference/pages/glossary.adoc

```diff
@@ -28,35 +35,48 @@ There, users can find knowledge base articles, support, announcements, and gener
 
-Elasticsearch:: Elasticsearch is a search engine based on the Lucene library. 
+Elasticsearch:: Elasticsearch is a search engine based on the Lucene library.
 OpenNMS persists/forwards certain data to Elasticsearch to store flow records and warehouse events and alarm history for long-term storage of this data.
 (See xref:operation:elasticsearch/introduction.adoc#elasticsearch[Elasticsearch Integration].)
 
-Flows:: Summaries of network traffic sent by network devices. 
+Events:: Structured historical records of occurrences in {page-component-title} and the nodes, interfaces, and services it manages.
+(See xref:operation:events/anatomy-events.adoc[Anatomy of an Event].)
+
+Flows:: Summaries of network traffic sent by network devices.
 (See xref:operation:flows/introduction.adoc[Flows].)
 
 Foreign Source:: A data source hosted outside of your local network.
 To customize monitoring of nodes in a foreign source, you can set a foreign source definition with your desired detectors and policies.
-(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign source definition].)
+(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign Source Definition].)
+
+Grafana:: Grafana is an open-source analysis and visualization web application.
+It monitors data sources and generates charts, graphs, and alerts.
+(See https://grafana.com/[Grafana].)
 
 Helm:: Helm is an an OpenNMS application for creating flexible dashboards to display and customize fault management and performance management data from Horizon and Meridian.
 Helm works with the Grafana analytics program to customize the data you can view.
 Not to be confused with the https://helm.sh/[Kubernetes Helm] package manager.
-(See https://docs.opennms.com/helm/latest[HELM documentation].)
+(See https://docs.opennms.com/helm/latest[HELM Documentation].)
 
 Horizon:: Open-source solution from OpenNMS that helps users visualize and monitor everything on their local and remote networks.
 Free, community-driven project includes the latest technology and features delivered through a rapid release cycle.
 
-IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services. 
+IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services.
 {page-component-title} can integrate with IFTTT to trigger actions on other services based on customized alarms.
 (See xref:operation:alarms/ifttt-integration.adoc[IFTTT Integration].)
 
+Instance:: A single, unique copy of a running program.
+If a program has multiple instances, that indicates that the program has been loaded into memory more than once.
+
+Jira:: Jira is a project management platform used by OpenNMS to track software issues, feature requests, and IT requests.
+(See https://en.wikipedia.org/wiki/Jira_(software)[Jira].)
```

Comment:
What about linking this to issues.opennms.com instead of wikipedia?

Reply:
Good idea, I can do that!